packages/cardpay-sdk/sdk/network-config-utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ethereumNetworks, gnosisNetworks, networkIds, polygonNetworks } from './constants';
+import { HubConfigResponse } from './hub-config';
+
+export const getWeb3ConfigByNetwork = (config: HubConfigResponse, network: string | number) => {
+  const networkName = (typeof network === 'number' ? networkIds[network] : network) as string;
+
+  if (ethereumNetworks.includes(networkName)) return config.web3.ethereum;
+  if (gnosisNetworks.includes(networkName)) return config.web3.gnosis;
+  if (polygonNetworks.includes(networkName)) return config.web3.polygon;
+
+  return { rpcNodeWssUrl: '', rpcNodeHttpsUrl: '' };
```

Comment:
Would it be better to throw an error here? I imagine it's undesirable to end up with empty config in case the network is unsupported

Reply:
Sure, that would be the next step, my goal first was just to use the same logic between both tools, but I guess I can already handle as an error here